files/configs/ingress/ingress-authserver-extensionservice.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: projectcontour.io/v1alpha1
```

Comment:
are those files supposed to be in here? looks like you based this PR off on the other PR instead of `development`

Reply:
Yea, I figured since this has to go in, that I'd simply base it off of prior PR. I guess best to do it from Development in future and just rebase ... Noted